configpm

```diff
@@ -225,6 +225,9 @@ i_time='define'
 prototype='define'
 EOT
 
+# Don't lie about PATCHLEVEL and SUBVERSION in %Config
+push @v_others, "PATCHLEVEL='$^V->{version}->[1]'\n";
+push @v_others, "SUBVERSION='$^V->{version}->[2]'\n";
```

Comment:
As already discussed, this gut-wrenching into the innards of the `$^V` object feels wrong. It would be better to `split()` to obtain these, I feel:

```perl
my ($ver_major, $ver_minor, $ver_patch) = split m/\./, $^V;

push @v_others, "PATCHLEVEL='$ver_patch'\n";
push @v_others, "SUBVERSION='$ver_minor'\n";
```

Though, now I look at it harder, those seem the wrong way round?

Reply:
Or just use `$Config{api_revision}` and friends

Reply:
the innards of version objects are well defined. I agree a method would have been better but we're 15 years too late for that.

https://metacpan.org/pod/distribution/version/lib/version/Internals.pod#Version-Object-Internals

Reply:
note that you also want to strip to leading `^v`
I reworked it as suggested via e661b0123fbcd85a31952fdbcb2b566c56d8026a